src/bsp/src/esp/src/BSP.cpp

```diff
@@ -1,17 +1,27 @@
 #include "BSP.h"
+#include "NetworkContainer.h"
 #include "driver/spi_slave.h"
+#include "esp_event.h"
 #include "esp_system.h"
 #include "hal/gpio_types.h"
 #include "hal/pin_map.h"
 #include "hal/spi_callbacks.h"
+#include "nvs_flash.h"
 #include <driver/gpio.h>
 
 BSP::BSP() = default;
 BSP::~BSP() = default;
 
 void BSP::initChip() {
+    // Init event loop
+    ESP_ERROR_CHECK(esp_event_loop_create_default());
+    // Init non volatile flash storage
+    ESP_ERROR_CHECK(nvs_flash_init());
```

Comment:
is the NVS used somewhere other than the init? There is no read or write on it

Reply:
It used internally by the network libraries of esp

Reply:
Maybe just add a comment